### ISSUE TITLE
Please include dark mode

### ISSUE THREAD

**Issue by shodanx2** [gwarble/EitherMouse#11]
Hi,

I use high contract dark mode theme called "grey eye.theme"

https://github.com/nitschis/GreyEveTheme

And I need to manually set the application to dark mode (I have the theme set, plus system and app windows dark mode enabled)

And this makes the application diffcult to read 

![image](https://github.com/user-attachments/assets/c86cd0a5-60da-473e-98da-f17997065627)

**Comment by gwarble** [comment]
Great idea to add a native dark mode setting, I will do that next time I'm making an update.  Thanks.

As for your current usage, it sounds like a "bug" with the program, which if i understand correctly is trying to force dark mode onto a program that doesn't support it...  I'm not sure why it would have trouble setting the background of my window to black

Edit: probably because i use a transparent control to allow dragging from anywhere in the window

long story short, i will look into both native dark mode and this "high contrast" mode
